### ISSUE TITLE
Improve the behavior of axes

### ISSUE THREAD

**Issue by NeatNit** [ChristopherChudzicki/math3d-react#205]
Just a short (read: long) list of things related to axes.

1. Make the z axis have default scale 1, same as the other two axes (#189)
2. Customizable tick spacing - currently it automatically tries to make useful ticks which works great, but occasionally the user might want to manually define the space between two ticks to a particular number (for example: pi/2). Also note that currently tick labels are currently rounded to 2 decimal places, see the image at the end to see why that's bad.
3. Make scale be internally multiplied by [max - min]. This is debatable! In my opinion setting all axes to scale = 1 should make all spheres look round. But if one axis's range is bigger than another, objects get squished. https://www.math3d.org/Ha8AUNMi
4. (discovered while typing up this issue) Bad behavior for ranges that don't include 0. The other two axes are rendered far away at the origin instead of gracefully being moved to within the scene. See image.
5. The "Size" option is badly named. It simply changes the size of the arrow head, but the name "Size" sounds like a synonym for "Scale". And under default settings, tweaking this is barely visible unless the number is made very large (because width is small). Basically what I'm saying is that I almost wrote here "the Size option doesn't change anything".

![image](https://user-images.githubusercontent.com/2400026/60214191-b0686680-986d-11e9-8d3b-59c4abac0ad0.png)

**Comment by prometeo2008** [comment]
Yes if you draw a semisphere of radius=4 sqrt(16-(x^2+y^2)), nearly it seems a flat cap. I don't know if you can define functions with plus and minus to form a complete sphere (±)

**Comment by prometeo2008** [comment]
Definitely you can't the page answer undefined symbol pm

**Comment by NeatNit** [comment]
> Yes if you draw a semisphere of radius=4 sqrt(16-(x^2+y^2)), nearly it seems a flat cap. I don't know if you can define functions with plus and minus to form a complete sphere (±)

With explicit surfaces, each (x,y) value can only have one corresponding z value. If you want to create a full sphere, there's two ways: parametric surface or implicit surface. Parametric surface is better because Math3D doesn't love implicit surfaces. It takes longer to process and the result is less detailed. See both in action here: https://www.math3d.org/cVdx5WVH

**Comment by prometeo2008** [comment]
The implicit is enough for me. Parametric looks better but you need to express the thing in a more complicated way and simplicity is one of the main beauties of Math3D

**Comment by prometeo2008** [comment]
Thanks for the answer. Do you think that parametric or implicit ways eliminate in any case the use of explicit functions with more than one value? I mean do you think these ways precludes, in any case, the need o using this kind of functions
